_field-types/supported-field-types/derived.md

```diff
@@ -0,0 +1,903 @@
+
+<details markdown="block">
+  <summary>
+    Response
+  </summary>
+  {: .text-delta}
+
+```json
+{
+  "took": 45,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 2,
+      "relation": "eq"
+    },
+    "max_score": 1.8754687,
+    "hits": [
+      {
+        "_index": "logs",
+        "_id": "1",
+        "_score": 1.8754687,
+        "_source": {
+          "request": "894030400 GET /english/images/france98_venues.gif HTTP/1.0 200 778",
+          "clientip": "61.177.2.0"
+        },
+        "fields": {
+          "request": [
+            "894030400 GET /english/images/france98_venues.gif HTTP/1.0 200 778"
+          ],
+          "clientip": [
+            "61.177.2.0"
+          ],
+          "url": [
+            "/english/images/france98_venues.gif"
+          ]
+        },
+        "highlight": {
+          "url": [
+            "/english/<em>images</em>/france98_venues.gif"
+          ]
+        }
+      },
+      {
+        "_index": "logs",
+        "_id": "3",
+        "_score": 1.8754687,
+        "_source": {
+          "request": "894250400 POST /english/venues/images/venue_header.gif HTTP/1.0 200 711",
+          "clientip": "61.177.2.0"
+        },
+        "fields": {
+          "request": [
+            "894250400 POST /english/venues/images/venue_header.gif HTTP/1.0 200 711"
+          ],
+          "clientip": [
+            "61.177.2.0"
+          ],
+          "url": [
+            "/english/venues/images/venue_header.gif"
+          ]
+        },
+        "highlight": {
+          "url": [
+            "/english/venues/<em>images</em>/venue_header.gif"
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+</details>
+
+## Performance
+
+Derived fields are not indexed but are computed dynamically by retrieving values from the `_source` field or doc values. Thus, they can be slow to execute. To improve performance, try the following:
+
+- Prune the search space by adding query filters on indexed fields together with derived fields.
+- Prefer using doc values over `_source` in the script for faster access, whenever applicable.
+- Consider using a [`prefilter_field`](#prefilter-field) to automatically prune the search space without explicit filters in the search request.
+
+### Prefilter field
+
+Specifying a prefilter field helps prune the search space automatically without adding explicit filters in the search request. The prefilter field specifies an existing indexed field (`prefilter_field`) on which to filter implicitly when constructing the query. The `prefilter_field` must be a text field (either [`text`]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/text/) or [`match_only_text`]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/match-only-text/)).
+
+For example, first update the mapping for the `method` derived field by specifying to prefilter on the `request` field: 
```

Comment:
"For example" doesn't work here because it's followed by an explicit instruction. Would something like "To try this" work instead?